extensions/wikia/ContentReview/tests/ContentReviewHelperTest.php

```diff
@@ -8,61 +8,119 @@ public function setUp() {
 	}
 
 	/**
-	 * @dataProvider replaceWithLastApprovedRevisionProvider
+	 * @param bool $isJsPage
+	 * @param bool $inNamespace
+	 * @param string $contentType
+	 * @param bool $expected
+	 * @dataProvider shouldPageContentBeReplacedData
+	 */
+	public function testShouldPageContentBeReplaced( $isJsPage, $inNamespace, $contentType, $expected ) {
+		$titleMock = $this->getMock( '\Title', [ 'isJsPage', 'inNamespace' ] );
+		$titleMock->expects( $this->once() )
+			->method( 'isJsPage' )
+			->willReturn( $isJsPage );
+
+		$titleMock->expects( $this->any() )
+			->method( 'inNamespace' )
+			->willReturn( $inNamespace );
```

Comment:
This doesn't check much, as it doesn't tell us anything about what the title is and what namespace is being checked. You could just add a `with` here, but as a small suggestion to abstract it a little more and allow explicit tests for things like https://wikia-inc.atlassian.net/browse/CE-2708, you could avoid mocking altogether given the methods used and just use `Title::makeTitle` so you're instead just controlling what the title is, as in this test:
https://github.com/Wikia/app/blob/3efa2cfe7144f/extensions/wikia/ProtectSiteJS/tests/ProtectSiteJSTest.php#L16

Up to you though. The `with` is needed at the very least to verify you're just checking the MW namespace.